src/components/jurisdiction/Jurisdiction.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <NestedSelect
+    :label="label"
+    :errorMessages="errorMessages"
+    :hideDetails="false"
+    :menuItems="jurisdictionOptions"
+    :value="jurisdiction ? jurisdiction.text : ''"
+    @change="emitChangeEvent($event)"
+  />
+</template>
+
+<script lang="ts">
+import { Component, Emit, Prop, Vue } from 'vue-property-decorator'
+import NestedSelect from './NestedSelect.vue'
+import { Location } from '@bcrs-shared-components/enums'
```

Comment:
Since this exported enum is added in this PR, it won't be found by storybook until this gets published. Once published there should be no issues.

I explain below why this is an exported enum and not internal.

Reply:
If you do the following, does it work?

https://github.com/bcgov/bcrs-shared-components/blob/58574e3808c35da695187c686f839c48c37584e5/README.md?plain=1#L50

Reply:
That did it!

Reply:
OK to resolve this conversation?